cmd/notation/verify.go

```diff
@@ -161,12 +172,31 @@ func resolveReference(ctx context.Context, opts *SecureFlagOpts, reference strin
 	return ref, nil
 }
 
-func printMetadataIfPresent(outcome *notation.VerificationOutcome) {
+func printResult(outputFormat, reference string, outcome *notation.VerificationOutcome) error {
+	output := verifyOutput{Reference: reference, Result: "Success", UserMetadata: map[string]string{}}
+
+	if reflect.DeepEqual(outcome.VerificationLevel, trustpolicy.LevelSkip) {
+		if outputFormat == ioutil.OutputPlaintext {
+			fmt.Println("Trust policy is configured to skip signature verification for" + reference)
+			return nil
+		} else {
+			output.Result = "Skipped"
+			return ioutil.PrintObjectAsJson(output)
+		}
+	}
+
 	// the signature envelope is parsed as part of verification.
 	// since user metadata is only printed on successful verification,
 	// this error can be ignored
 	metadata, _ := outcome.GetUserMetadata()
 
+	if outputFormat == ioutil.OutputJson {
+		output.UserMetadata = metadata
+		return ioutil.PrintObjectAsJson(output)
+	}
```

Comment:
are we throwing away outputFormat object if output != json ?

Reply:
yes, can make it cleaner and only setup the object if output format is json